### PR TITLE
Update a greeting for french

### PR DESCRIPTION
**Pull request by Alexxx1407**
Uh just a thought i had!

### CODE REVIEW

File: Monika After Story/game/script-greetings.rpy [hunk 1]
```diff
@@ -2119,7 +2119,24 @@ label greeting_french:
     m 1hub "Ahaha!"
     m 3hksdlb "I'm practicing some French. I just told you that you have very beautiful eyes~"
     m 1eka "It's such a romantic language, [player]."
-    m 1hua "Maybe both of us can practice it sometime, mon amour~"
+    m 4esa "Do you speak French [player]{nw}?"
+    $ _history_list.pop()
+    if persistent._mas_pm_lang_other is not False:
+        menu:
+            m "Do you speak French [player]?"
```
Comment (lunulae) [inline Monika After Story/game/script-greetings.rpy:2126]:
```suggestion
            m "Do you speak French, [player]?"
```

File: Monika After Story/game/script-greetings.rpy [hunk 2]
```diff
@@ -2119,7 +2119,24 @@ label greeting_french:
...
+            "Yes.":
+                m 2wud "Oh!"
+                m 1rkbsc "Maybe you could...{w=0.4}teach me one day?"
+                m 5tkbsu "I'm sure you will. I love you [player]."
```
Comment (lunulae) [inline Monika After Story/game/script-greetings.rpy:2130]:
```suggestion
                m 5tkbsu "I'm sure you will. I love you, [player]."
```

File: Monika After Story/game/script-greetings.rpy [hunk 3]
```diff
@@ -2119,7 +2119,24 @@ label greeting_french:
...
+
+            "No.":
+                m 1eua "That's alright!"
+                m 2eua "I'm not a good at french too!"
```
Comment (lunulae) [inline Monika After Story/game/script-greetings.rpy:2134]:
```suggestion
                m 2eua "I'm not so skilled at French either."
```

File: Monika After Story/game/script-greetings.rpy [hunk 4]
```diff
@@ -2119,7 +2119,24 @@ label greeting_french:
...
+                m 3eublb "We will learn it together one day!"
```
Comment (lunulae) [inline Monika After Story/game/script-greetings.rpy:2135]:
```suggestion
                m 3eublb "But that just means we could learn it together one day!"
```

File: Monika After Story/game/script-greetings.rpy [hunk 5]
```diff
@@ -2119,7 +2119,24 @@ label greeting_french:
...
+
+            "Oui oui baguette.":
+                m 2efo "Uh...{w=0.4}are you okay [player]?"
+                m 1efw "I'll skip that!"
```
Comment (lunulae) [inline Monika After Story/game/script-greetings.rpy:2139]:
```suggestion
                m 1efw "Maybe I'll pass on those lessons, ehehe~"
```

**Review by lunulae — COMMENTED** [review]
Mostly small grammar things, and a few rephrasings for natural flow. Altogether though I like this, great work ❤️

**Comment by jmwall24** [comment]
Closing as we want to avoid menus in greetings. We have plans to add ways to tell her what languages you speak in the future and we'll adjust the greets to reflect it at that time.

**Pull request closed without merging**